### PR TITLE
[doc] Add examples of SortingHat API via Python

### PR DESCRIPTION
**Pull request by animeshk08**
Fixes: #284 
This commit add examples of SortingHat API via Python.
A new section Basic API calls was added to include the
example.

Signed-off-by: Animesh Kumar <animuz111@gmail.com>

### CODE REVIEW

**Comment by animeshk08** [comment]
Hi @valeriocos. Please let me know if any improvements are needed.

Also are there any 'readthedocs' available for sortinghat I was thinking of linking them to the APIs.

The README is getting pretty big I think we can re-arrange it similar to the one at [sirmordred](https://github.com/chaoss/grimoirelab-sirmordred/blob/master/README.md). WDYT?

**Comment by valeriocos** [comment]
Thank you @animeshk08 , I'm on it

File: README.md [hunk 1]
```diff
@@ -345,6 +345,212 @@ $ sortinghat load identities.json
...
+```
+api.search_profiles(db, no_gender=False)
+```
+
+* Search for unique identities.
+
+The API returns a list of unique identities. The term will be compared with name, email, username and source values
+of each identity. When `source` is given, this search will be only performed on identities linked to this source.
+```
+api.search_unique_identities(db=db,term='example',source='scm')
```
Comment (valeriocos) [inline README.md:536]: missing spaces between params

**Comment by valeriocos** [comment]
No worries @animeshk08 ! :)

> I think we can add a subheading 'Key terms' under Basic API Calls and mention the necessary terms there. WDYT?

Yes, good idea! Thanks!

**Comment by animeshk08** [comment]
Hi @valeriocos. I have made the changes please have a look :)

**Pull request merged**